### DISCUSSION TITLE
Hosting With Netlify or Heroku

### DISCUSSION THREAD

**Discussion by FashMuyhee** [adonisjs/core#1816] [Help]
Hello guys,
I have an api i built with adonisjs , i want know if it is possible to host on either heroku or netlify with the MySQL database

**Comment by junicodes** [comment]
Yes you can host it on heroku ...I am sure about that but I am not of netlify

**Reply by FashMuyhee** [reply]
oh, i thought netlify has the option for database hosting.
Thanks 🙏🙏

**Comment by junicodes** [comment]
There is a database on heroku called clearDB...it will give you storage support.

**Reply by FashMuyhee** [reply]
I'll look it up

**Comment by thetutlage** [comment]
Netlify is for static sites hosting only (if I am knowledge is upto date).  You can also try cleavr https://cleavr.io/adonis, they have first class support for AdonisJS apps.

**Reply by junicodes** [reply]
Cleavr is very good too.

**Comment by junicodes** [comment]
When hosting to heroku only, for ease try not to ignore you .env file and change your Host to 0.0.0.0 also replace your database configuration to your presumed created clearDb database on heroku (you can only create this DB after adding billing details).
